src/App.tsx

```diff
@@ -1,10 +1,67 @@
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
+import React, { useEffect, useMemo, useState } from 'react';
 import { UserWarning } from './UserWarning';
+import { TodoInfo } from './components/TodoInfo';
+import { Error } from './components/Error';
+import { Header } from './components/Header';
+import { Footer } from './components/Footer';
+import { client } from './utils/fetchClient';
+import { Todo } from './types/Todo';
+import { Filter } from './enums/enums';
 
-const USER_ID = 0;
+const USER_ID = 10567;
+
+const URL = `/todos?userId=${USER_ID}`;
 
 export const App: React.FC = () => {
+  const [todos, setTodos] = useState<Todo[]>([]);
+  const [filterValue, setFilterValue] = useState(Filter.All);
+  const [isAddError, setIsAddError] = useState(false);
+  const [isDeleteError, setIsDeleteError] = useState(false);
+  const [isUpdateError, setIsUpdateError] = useState(false);
+
+  useEffect(() => {
+    const fetchData = async () => {
+      try {
+        const response = await client.get(URL);
+        const todosData = response as Todo[];
+
+        setTodos(todosData);
+      } catch (error) {
+        setIsAddError(true);
```

Comment:
Here's the problem - in the case of the failed request on **loading** todos, you will get all three errors
<img width="397" alt="Screenshot 2023-06-12 at 20 49 48" src="https://github.com/mate-academy/react_todo-app-loading-todos/assets/32853034/572f574a-1fd9-4c90-9686-5fda4e307383">

There should be only one error showing that loading is failed